coordinator/tests/resources/test-helpers.ts

```diff
@@ -27,3 +27,18 @@ export function addEmptyCommunicationRequestToBundle(bundle: fhir.Bundle): void
     payload: []}
   bundle.entry.push({resource: communicationRequest})
 }
+
+declare global {
```

Comment:
Helper method extending the Array class in typescript to keep code clean in a test. This is introduced as a necessity because we're populating our test data from our examples. Could be an argument to remove the dependency here and use the minimal examples you've been working on, but again, see this as a wider refactoring piece of work